packages/financial-templates-lib/src/clients/FinancialContractClient.js

```diff
@@ -251,6 +251,7 @@ class FinancialContractClient {
           .mul(this.latestCumulativeFundingRateMultiplier)
           .div(this.fixedPointAdjustment)
           .toString(),
+        rawTokens: position.tokensOutstanding.toString(),
```

Comment:
so the `numTokens`, considering the funding rate, is still used when the bot decided if a position is liquidatable, correct? then, if a position deemed liquidatable the `rawTokens` is liquidated. This way you still factor in the funding rate to the liquidation decision but use the `rawTokens` (the actual amount in the position) when doing the liquidation, right?

Also could you please add a comment here to explain this distinction between `numTokens` and `rawTokens`.

Reply:
> so the numTokens, considering the funding rate, is still used when the bot decided if a position is liquidatable, correct?

Correct.

Added comment:
```
numTokens: this.toBN(position.tokensOutstanding.toString()) 
          .mul(this.latestCumulativeFundingRateMultiplier)
          .div(this.fixedPointAdjustment)
          .toString(),
        // Applies the current funding rate to the sponsor's raw amount of tokens outstanding. `numTokens` is equal
        // to `rawTokens * cumulativeFundingRateMultplier`, and this client uses `numTokens` to determine whether
        // a position is liquidatable or not. If a position is liquidatable, then `rawTokens` amount of debt needs
        // to be repaid by the liquidator to liquidate the position.
        rawTokens: position.tokensOutstanding.toString(), // Raw tokens is the `tokensOutstanding` parameter in the
        // on-chain position struct. It represents the actual amount of tokens outstanding for a sponsor and is the
        // maximum amount of tokens needed to fully liquidate a position. The liquidator bot therefore uses this
        // amount.
```